lib/logstash/outputs/elasticsearch.rb

```diff
@@ -103,6 +103,11 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   # not also set this field. That will raise an error at startup
   config :path, :validate => :string
 
+  # Pass a set of key value pairs as the URL query string. This query string is added
+  # to every host listed in the 'hosts' configuration. If this list contains urls which
+  # already contain a query string, this one will be appended to the existing one.
```

Comment:
Wait, why would someone put a full URL in here? This sentence "If this list contains urls which
 +# already contain a query string, this one will be appended to the existing one." is very confusing to me.